test/controllers/static_pages_controller_test.rb

```diff
@@ -76,7 +76,7 @@ class StaticPagesControllerTest < ActionDispatch::IntegrationTest
     assert_response :success
     # assert_template 'home'
     # Check that it has some content
-    assert_includes @response.body, 'les projets de logiciel libre'
+    assert_includes @response.body, 'les projects de logiciel libre'
```

Comment:
No. "projets" is correctly spelled (in French).

Reply:
Makes sense to me. I agree, it'd be better to have "ignore specific lines", but hopefully we can spell "projects" correctly most of the time :-).